index.html

```diff
@@ -3867,9 +3914,121 @@ <h3>Parsing content</h3>
       </li>
       <li>
         Return |record|.
+        <p class="note">
+          Applications may call <code>toRecords()</code> on <a>data</a> to
+          parse it to <a>NDEF records</a>, or may parse it themselves.
+        </p>
       </li>
     </ol>  <!-- parsing NDEF smart-poster record -->
   </div>
+  <div>
+    To <dfn>parse a smart-poster size record</dfn> given an |ndefRecord| into a
+    |record:NDEFRecord|, run these steps:
+    <ol class=algorithm data-link-for="NDEFRecord">
+      <li>
+        Set |record|'s <a>recordType</a> to "`s`".
```

Comment:
Shouldn't it be `:s`?

Reply:
Same for type and action record.

Reply:
Right. And we need an extra algo for inserting `:` before local types.